birdhouse/optional-components/stac-data-proxy/config/magpie/permissions.cfg.template

```diff
@@ -0,0 +1,12 @@
+# NOTE:
+# Assume 'secure-data-proxy' would exist if needed.
+# Since 'secure-data-proxy' < 'stac-data-proxy', it should be loaded first.
```

Comment:
The use of `<` here is confusing. Do you mean that one depends on the other? One overrides variable definitions of the other one?

Reply:
Sorting order by file name. Literally `__lt__` used by `sorted()`.

Reply:
I've updated the description to be more explicit. Let me know if it is sufficient.